src/Symfony/Component/PropertyInfo/Tests/Extractor/ReflectionExtractorTest.php

```diff
@@ -213,6 +213,25 @@ public function testExtractPhp71TypeWithParentConstructor()
         $this->assertEquals($type, $this->extractor->getTypes('Symfony\Component\PropertyInfo\Tests\Fixtures\Php71DummyChild3', $property, array()));
     }
 
+    /**
+     * @dataProvider defaultValueProvider
+     */
+    public function testExtractWithDefaultValue($property, $type)
+    {
+        $this->assertEquals($type, $this->extractor->getTypes('Symfony\Component\PropertyInfo\Tests\Fixtures\DefaultValue', $property, array()));
```

Comment:
`DefaultValue::class`

Reply:
Done, but there are a lot of FQN refers in the test class. It's ok?